cmd/broker/broker_suite_test.go

```diff
@@ -127,6 +127,12 @@ func NewBrokerSuiteTest(t *testing.T, version ...string) *BrokerSuiteTest {
 	return NewBrokerSuiteTestWithConfig(t, cfg, version...)
 }
 
+func NewBrokerSuiteTestWithConvergedRegionMappings(t *testing.T, version ...string) *BrokerSuiteTest {
```

Comment:
ConvergedRegion is misleading and inconsistent with our naming convention. 
ConvergedCloudRegion.